app/controllers/procedures_controller.rb

```diff
@@ -1,58 +1,59 @@
 class ProceduresController < ApplicationController
-     #Currently shows all Procedures
-    def index
-        @procedures = Procedure.all
-    end
+  before_action :authenticate_user!, only: [:edit, :update, :destroy, :base_params, :create_params, :update_params]
+  #Currently shows all pending Procedures
+  def index
+    @procedures = Procedure.where("status = 'approved'")
+  end
     
-    #Creates a new Procedure
-    def new
-        @procedure = Procedure.new
-        @facilities = Facility.all
-    end
+  #Creates a new Procedure
+  def new
+    @procedure = Procedure.new
+  end
 
-    #adds new procedure to database
-    def create
-        procedure = Procedure.new
-        procedure.name = params[:procedure][:name]
-        procedure.date = params[:procedure][:date]
-        procedure.description = params[:procedure][:description]
-        procedure.price = params[:procedure][:price]
-        procedure.facility_id = params[:procedure][:facility_id]
-        procedure.status = "pending"
-        procedure.save
-        redirect_to "/"
-    end
+  #adds new procedure to database
+  def create
+    @procedure = Procedure.create(create_params)
+    @procedure.status = "pending"
+    @procedure.save
+    redirect_to "/"
+  end
 
-    #edit exisiting database entry -- only available to admin? -------
-    def edit
-        @procedure = Procedure.find(params[:id])
-        @facilities = Facility.all
-    end
+  #edit exisiting database entry -- only available to admin -------
+  def edit
+    @procedure = Procedure.find(params[:id])
+  end
 
-    #update exisiting database entry -- only available to admin? -------
-    def update
-        procedure = Procedure.new
-        procedure.name = params[:procedure][:name]
-        procedure.date = params[:procedure][:date]
-        procedure.description = params[:procedure][:description]
-        procedure.price = params[:procedure][:price]
-        procedure.facility_id = params[:procedure][:facility_id]
-        procedure.status = params[:status] #drop down?
-        procedure.save
-        redirect_to "/"
-    end
+  #update exisiting database entry -- only available to admin -------
+  def update
+    @procedure = Procedure.find(params[:id])
+    @procedure.update(update_params)
+    @procedure.status = 'pending'
+    @procedure.save
+    redirect_to "/admins"
```

Comment:
use the path helper method for admins

Reply:
Changed "/admins" to "admins_path" in admins, items, facilities and procedures controllers